aws-auth/src/main/java/software/amazon/awssdk/awsauth/credentials/HttpCredentialsProvider.java

```diff
@@ -132,7 +132,7 @@ public void close() {
         private String asyncThreadName;
 
         /**
-         * Created using {@link #builder()}.
+         * Created using {@link #build()}.
```

Comment:
Nitpicky but this seemed more correct before. Comment is mostly useless though anyways.

Reply:
yup, removed